reproducibility_project/clean_by_engine.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+echo Removing jobs from all engines but $1
+for ENGINE in cassandra mcccs gomc gromacs hoomd lammps-VU lammps-UD
+do
+    if [[ $ENGINE != $1 ]]
+    then
+        signac rm $(signac find engine $ENGINE)
+        echo Removed jobs for $ENGINE
+    fi
+done
```

Comment:
I would either add this to the lrc_cutoff_subproject, or allow an option to specify which subproject you want to filter other engine jobs from.

Reply:
That or you could make a `bin` folder at `reproducibility_study/reproducibility_project/bin` and put these scripts in here. Then the user can just run `../bin/clean_by_engine.sh my_engine` if they are in the sub project.